### PR TITLE
fix: devtool validation too lenient

### LINKED ISSUE #86: devtool validation is too lenient?
I tried patterns like `eval-source-map-foo` and those passed. Worth looking into at least.

### PR DESCRIPTION
**Pull request by nlarche**
closes #86

### CODE REVIEW

File: src/properties/output/index.test.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ const validModuleConfigs = [
   { crossOriginLoading: 'anonymous' },
   // #17
   { hashDigestLength: 6 }, // undocumented
-
+  // #18
+  { publicPath: '' },
```
Comment (kentcdodds) [inline src/properties/output/index.test.js]: This shouldn't be in this PR. Can you clean up your commit history to only have relevant changes?

Comment (nlarche) [inline src/properties/output/index.test.js]: Yes ok, I clean up my commit

**Comment by nlarche** [comment]
@kentcdodds, it should be ok

**Comment by kentcdodds** [comment]
LGTM

**Comment by jonathanewerner** [comment]
👍🏻

**Pull request merged**